src/angular/ghettobox/ghettobox.scss

```diff
@@ -0,0 +1,96 @@
+@import '../styles/common';
+
+.sbb-ghettobox {
+  display: flex;
+  background-color: $sbbColorIron;
+  color: $sbbColorWhite;
+  padding: pxToRem(16);
+  justify-content: center;
+  align-items: flex-start;
+  font-size: pxToRem($sizeFontDefault);
+  line-height: pxToRem($sizeLineHeightDefault);
+
+  @include mq($from: tablet) {
+    padding: pxToRem(36) pxToRem(24) pxToRem(36) pxToRem(48);
+  }
+  @include mq($from: desktop) {
+    padding: pxToEm(40) pxToEm(56);
+  }
+  @include mq($from: desktop4k) {
+    font-size: pxToRem($sizeFontDefault4k);
+    line-height: pxToRem($sizeLineHeightDefault4k);
+  }
+  @include mq($from: desktop5k) {
+    font-size: pxToRem($sizeFontDefault5k);
+    line-height: pxToRem($sizeLineHeightDefault5k);
+  }
+
+  &.sbb-ghettobox-link {
+    text-decoration: none;
+    transition: background-color 0.3s cubic-bezier(0.785, 0.135, 0.15, 0.86);
+
+    &:hover,
+    &:focus,
+    &:focus-within,
+    &:active {
+      background-color: $sbbColorBlack;
+      color: $sbbColorWhite;
+    }
+  }
+}
+
+.sbb-ghettobox-icon,
+.sbb-ghettobox-close {
```

Comment:
for me it's not clear why text-align: right only above 4k is used and why above 4k fles is "1 1"